sycl/test-e2e/Matrix/SG32/get_coordinate_ops.cpp

```diff
@@ -15,9 +15,6 @@
 // REQUIRES: aspect-ext_intel_matrix
 // REQUIRES-INTEL-DRIVER: lin: 30049, win: 101.4943
 
-// XFAIL: arch-intel_gpu_pvc
-// XFAIL-TRACKER: GSD-10524
-
```

Comment:
hmm, it seems you need to rebase this PR.
I don't see this xfail in the repo: https://github.com/intel/llvm/blob/sycl/sycl/test-e2e/Matrix/SG32/get_coordinate_ops.cpp